scripts/version.js

```diff
@@ -138,7 +171,16 @@ async function run() {
       chalk.green(`  Updated react-router-native to version ${version}`)
     );
 
-    // 6. Commit and tag
+    // 6. Update react-router and react-router-dom versions in the examples
```

Comment:
I'm torn on doing this in 1 or 2 steps. Leaning towards 1 function due to only then only having to read the filesystem once, but then it breaks out from the current pattern of each function having a single thing to do.. unless we grab all the examples when we initially run version and pass them to the 2 functions 🤔